Html/index.html

```diff
@@ -0,0 +1,486 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Please, I'd suggest you use  your name, this would help make your site look more visually impressive. Please kindly fix this 🙏